scripts/gulp/icons.js

```diff
@@ -27,6 +36,23 @@ function pfIconFont() {
         timestamp: Math.round(Date.now() / 1000)
       })
     )
+    // gulp-iconfont emits a 'glyphs' event
+    // we use to update existingIconCodes file
+    .on('glyphs', function(glyphs, options) {
+      const newIconsObj = glyphs.reduce((iconsObj, glyph) => {
+        const { name, unicode } = glyph;
+        const codePoint = unicode[0].codePointAt(0);
+        iconsObj[name] = codePoint.toString(16).toUpperCase();
+        iconsObj.codePoints.push(codePoint);
+        return iconsObj;
+      }, { codePoints: [] });
+
+      fs.writeFile(
+        path.join(process.cwd(), '/src/icons/existingIconCodes.json'),
+        JSON.stringify(newIconsObj),
+        err => err && console.log(err)
```

Comment:
Remove this line. If there's an error it will already be printed AND it should crash the program as well.

Reply:
FYI it looks like the program fails to run if I remove this line completely (node complains that a callback is expected here), but I'll remove the console.log.

Reply:
Ah, just use `fs.writeFileSync` instead!